spec/services/giphy/image_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe Giphy::Image do
+  describe "#giphy_img?" do
```

Comment:
This was a leftover :)

Reply:
Sorry about that @rhymes, I took care of it!